content/src/fieldConfig/business-formation.json

```diff
@@ -153,7 +152,10 @@
         "tableHeader": "Name,Address,Sign*,Actions",
         "titleLabel": "Title",
         "description": "Authorized Representative(s) should sign below by adding their full name. You can have up to 10 signers listed.",
-        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer."
+        "errorInlineSignerName": "Enter a signer's name.",
+        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer.",
+        "errorInLineNameAndSignature": "Enter a signer’s name and click the checkbox to authorize their signature.",
+        "errorInLineSignature": "Click the checkbox to authorize the signer’s signature."
```

Comment:
Can we update `InLine` so the `'I'` is lower case instead of a capital letter: `'L'`?

Reply:
oh yes, good catch, I have been looking at it for so long, didn't notice the difference.